From fb126eabbce4ce8ad4a14d52c61c2726a16c6d5f Mon Sep 17 00:00:00 2001
From: yvanliang <yvan6liang@outlook.com>
Date: Fri, 9 Oct 2026 15:43:59 +0800
Subject: [PATCH 3/4] GSI Android 17 compatibility:
 0003-sepolicy-label-dmabuf-filesystem-to-fix-camera-video-recording.patch

Change-Id: I08f79d47b04c675d2e221e3fefa0dd295baa9e90
---
 private/genfs_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/private/genfs_contexts b/private/genfs_contexts
index 8bbbfd8..3f2cc29 100644
--- a/private/genfs_contexts
+++ b/private/genfs_contexts
@@ -123,6 +123,7 @@ genfscon fusectl / u:object_r:fusectlfs:s0
 genfscon selinuxfs / u:object_r:selinuxfs:s0
 genfscon cgroup / u:object_r:cgroup:s0
 genfscon cgroup2 / u:object_r:cgroup_v2:s0
+genfscon dmabuf / u:object_r:rootfs:s0
 # sysfs labels can be set by userspace.
 genfscon sysfs / u:object_r:sysfs:s0
 genfscon sysfs /bus/thunderbolt/devices           u:object_r:sysfs_thunderbolt:s0
